src/ecs/systems/server/HealthUpdate.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+** EPITECH PROJECT, 2022
+** RTYPE
+** File description:
+** playerHealthUpdate
+*/
+
+#pragma once
+
+#include <functional>
+#include <iostream>
+#include "Constant.hpp"
+#include "Entity.hpp"
+#include "World.hpp"
+#include "components/NetworkId.hpp"
+#include "components/Position.hpp"
+
+namespace ecs::systems
+{
+    /**
+     * System used to push death of entity message to the server
+     * outgoing queue when is life as gone under 0.
+     */
+    std::function<void(World &)> playerHealthUpdate = [](World &world) {
```

Comment:
File and system must have the same name, the only difference being the first letter capitalization